salt/_modules/metalk8s_cordon.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+from salt.exceptions import CommandExecutionError
+
+import metalk8s_kubernetes
+
+try:
+    import kubernetes  # pylint: disable=import-self
+    import kubernetes.client
+    from kubernetes.client.rest import ApiException
+    from urllib3.exceptions import HTTPError
+
+    # Workaround for https://github.com/kubernetes-client/python/issues/376
+    from kubernetes.client.models.v1beta1_custom_resource_definition_status import V1beta1CustomResourceDefinitionStatus
+
+    def set_conditions(self, conditions):
+        if conditions is None:
+            conditions = []
+        self._conditions = conditions
+
+    setattr(
+        V1beta1CustomResourceDefinitionStatus, 'conditions',
+        property(
+            fget=V1beta1CustomResourceDefinitionStatus.conditions.fget,
+            fset=set_conditions
+        )
+    )
+    # End of workaround
```

Comment:
I think this workaround is not needed here

Reply:
Seen f2f - the workaround is for most listing operations but operates by modifying an underlying operation.

Reply:
Can be removed since that correction is only for CRDs. It may arise for other listing endpoints though.